browser/index.ts

```diff
@@ -68,18 +68,20 @@ const generatePdf = async (url: string) => {
     );
   }
 
+  const { headerTemplate, footerTemplate } =
+    getHeaderandFooterTemplates(template);
+
+  console.log({ headerTemplate, footerTemplate });
```

Comment:
We should add some context logger which could be turned on/off on production.

Reply:
There is a logger this was just a leftover to delete